res/build.sh

```diff
@@ -51,3 +73,8 @@ pdflatex -output-directory=book/latex/ \
          -jobname=handbook-a4 \
          book/latex/handbook.tex \
          >/dev/null
+
+## Remove temporary font directory
+
+rm -rf $TMP_FONTDIR
```

Comment:
doesn't this break if the user already has their own `~/.fonts`?

Reply:
This should only remove the temporary directory within `~/.fonts`, shouldn't it? Not the entire `~/.fonts` hierarchy. i have a `~/.fonts` directory, and the script certainly didn't trash it. The line:

    TMP_FONTDIR=$(mktemp -d -p ~/.fonts)

creates a temporary directory *within* `~/.fonts/`.

Reply:
Oh, I had the wrong idea about the semantics! It's good, then.